manifests/d/DisplayLink/GraphicsDriver/11.0.2412.0/DisplayLink.GraphicsDriver.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: DisplayLink.GraphicsDriver
+PackageVersion: 11.0.2412.0
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: -silent
+  SilentWithProgress: -silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://synaptics.com/sites/default/files/exe_files/2023-01/DisplayLink%20USB%20Graphics%20Software%20for%20Windows11.0%20M0-EXE.exe
+  InstallerSha256: 14391CA9482AB506026D588990FA1E3F06846B893D639D902BC065EB696E480D

```

Comment:
```suggestion
```

Reply:
It appears that the developer has removed 32-bit support from this package: 
![image](https://user-images.githubusercontent.com/114952565/214661782-c29c54fd-3702-43a8-b16b-926de4899451.png)